### PR TITLE
Events Page Dynamic Rendering

### PR DESCRIPTION
**Pull request by hecwon**
Events page now dynamically rendering sample data, with a working implementation of map. Implement map to have markers that dynamically render coordinates.

### CODE REVIEW

**Review by KateFeaster — CHANGES_REQUESTED** [review]
Looks fantastic! Awesome to see.

Two tiny changes then good to go!

File: frontend/components/Events.jsx [hunk 1]
```diff
@@ -1,12 +1,53 @@
...
-import { View, Text } from 'react-native'
+import React, { useState } from 'react';
+import { SafeAreaView, ScrollView, StyleSheet, Text, View, Pressable } from 'react-native';
+import eventData from '../sampleData/events.js';
+import EventList from '../components/events/EventList.jsx';
+import Map from '../components/events/Map.jsx';
 
 const Events = () => {
+  const [tab, setTab] = useState('attending');
+  console.log(`${tab} was pressed`);
```
Comment (kallycao) [inline frontend/components/Events.jsx:9]: Please remove this console.log before merging

**Review by kallycao — COMMENTED** [review]
Great work Hector! Looks solid. Since there are pending requests from Kate, I will not be able to approve

**Review by KateFeaster — APPROVED** [review]
No comments. Looks great!

**Review by kallycao — APPROVED** [review]
Amazing job Hector! Thank you for providing sample data and tackling geolocation. You're a beast.

**Pull request merged**